framework/wazuh/core/cluster/utils.py

```diff
@@ -18,15 +18,80 @@
 
 from wazuh.core import common, pyDaemonModule
 from wazuh.core.configuration import get_ossec_conf
-from wazuh.core.exception import WazuhException, WazuhError, WazuhInternalError
+from wazuh.core.exception import WazuhError, WazuhException, WazuhInternalError
 from wazuh.core.results import WazuhResult
 from wazuh.core.utils import temporary_cache
 from wazuh.core.wazuh_socket import create_wazuh_socket_message
 from wazuh.core.wlogging import WazuhLogger
 
+NO = 'no'
+YES = 'yes'
+DISABLED = 'disabled'
+HAPROXY_HELPER = 'haproxy_helper'
+HAPROXY_DISABLED = 'haproxy_disabled'
+HAPROXY_ADDRESS = 'haproxy_address'
+HAPROXY_PORT = 'haproxy_port'
+HAPROXY_PROTOCOL = 'haproxy_protocol'
+HAPROXY_USER = 'haproxy_user'
+HAPROXY_PASSWORD = 'haproxy_password'
+HAPROXY_BACKEND = 'haproxy_backend'
+HAPROXY_RESOLVER = 'haproxy_resolver'
+FREQUENCY = 'frequency'
+EXCLUDED_NODES = 'excluded_nodes'
+AGENT_CHUNK_SIZE = 'agent_chunk_size'
+AGENT_RECONNECTION_TIME = 'agent_reconnection_time'
+AGENT_RECONNECTION_STABILITY_TIME = 'agent_reconnection_stability_time'
+IMBALANCE_TOLERANCE = 'imbalance_tolerance'
+REMOVE_DISCONNECTED_NODE_AFTER = 'remove_disconnected_node_after'
+
 logger = logging.getLogger('wazuh')
 execq_lockfile = os.path.join(common.WAZUH_PATH, "var", "run", ".api_execq_lock")
 
+def validate_haproxy_helper_config(helper_config: dict) -> dict:
+    """Validate HAProxy helper configuration section.
+
+    Parameters
+    ----------
+    helper_config : dict
+        Configuration to validate.
+
+    Returns
+    -------
+    dict
+        Validated configuration for HAProxy Helper.
+
+    Raises
+    ------
+    WazuhError (3004)
+        If some value have an invalid type.
+    """
+    if helper_config[HAPROXY_DISABLED] == NO:
+        helper_config[HAPROXY_DISABLED] = False
+    elif helper_config[HAPROXY_DISABLED] == YES:
+        helper_config[HAPROXY_DISABLED] = True
+
+    for field in [
+        FREQUENCY,
```

Comment:
If we want to be stricter, we could validate that the port is between 0 and 65535.

```suggestion
    for field in [
        HAPROXY_PORT,
        FREQUENCY,
```